### PR TITLE
Documentation for new sensor FinTS

### PR DESCRIPTION
**Pull request by ChristianKuehnel**
**Description:**
This PR adds an initial documentation for the new sensor FinTS.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14334

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.fints.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+With the FinTS sensor, you can fetch your account information from your bank. This only works with banks that support the FinTS (aka. HBCI) standard. The FinTS standard used by many German banks. So if you do not have a German bank account, this will most likely not work for you. To find out if your bank supports FinTS, check the bank's website or call their hotline.
+
+For each account you have with the bank, a separate sensor is created. If you have several accounts with a bank, you can select which ones you want to have, and you can also give the accounts a name.
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+sensor:
+ - platform: fints
+    name: <give the bank a name>
```
Comment (frenck) [inline source/_components/sensor.fints.markdown]: `name` seems to be an optional configuration variable.

Please remove all optional parameters from initial example. We'd like to keep the first example as minimal as possible.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks, @ChristianKuehnel, left you a minor comment. Could you please take a look? 👍

File: source/_components/sensor.fints.markdown (hunk 1 above)
Comment (ChristianKuehnel) [reply source/_components/sensor.fints.markdown]: done

Comment (frenck) [reply source/_components/sensor.fints.markdown]: @ChristianKuehnel It is still in?

Comment (ChristianKuehnel) [reply source/_components/sensor.fints.markdown]: Hm, yes sorry. But now its in...

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @ChristianKuehnel!

Can be merged as soon as the parent PR gets merged.

**Pull request merged**